client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -73,10 +86,22 @@ const TransferPage = ( props: TransferPageProps ): JSX.Element => {
 					buttonText={ __( 'Continue' ) }
 					// translators: Transfer a domain to another user
 					headerText={ __( 'To another user' ) }
-					mainText={ __( 'Transfer this domain to any administrator on this site' ) }
+					mainText={ mainText }
 				/>
-				<div className="transfer-page__item-separator"></div>
+			);
+		}
+
+		if ( ! ( isPrimaryDomain && isAtomic ) ) {
+			if ( options.length > 0 ) {
+				options.push( <div key="separator" className="transfer-page__item-separator"></div> );
+			}
+			const mainText = isMapping
+				? __( 'Transfer this domain connection to any site you are an administrator on' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( 'Transfer this domain to any site you are an administrator on' )` *ES Score: 7*